compoments/report-repair/contact-us.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {serviceName} from '../../helpers/constants';
+
+const ContactUs = () => {
+  const title = 'You need to contact us to report this repair'
+  return (
+    <div className="govuk-grid-row govuk-body-m">
+      <header>
+        <title>{title} - {serviceName}</title>
+      </header>
+      <div className="govuk-grid-column-two-thirds">
+        <h1 className='govuk-heading-xl'>{title}</h1>
+        <div className="govuk-inset-text">
+          Contact us via telephone 01636 650 000
```

Comment:
Will this phone number need to be editable?

Reply:
Yes, it would need to be configurable.
If this PR is merged prior to #33, will ensure the other PR updates the telephone number and vice versa too.